lib/Runtime/Library/TypedArray.h

```diff
@@ -272,6 +279,22 @@ namespace Js
             return GetLibrary()->GetUndefined();
         }
 
+        inline Var BaseTypedDirectGetItemNoDetachCheck(__in uint32 index)
```

Comment:
>BaseTypedDirectGetItemNoDetachCheck [](start = 19, length = 35)

likewise here, will be good to add an assert for detachedbuffer in all *NoDetachCheck() functions.